File: homework_1/1.max_of_three/max_of_three.cpp
#include <iostream>
using namespace std;

int main(){
    int x, y, z, max;
    cin >> x >> y >> z;

    max = x;
    if(y > max)
        max = y;
    if(z > max)
        max = z;
    
    cout << max << endl;

    return 0;
}

File: homework_1/1.max_of_three/max_of_three.java
import java.util.Scanner;

class max_of_three {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int x = input.nextInt();
        int y = input.nextInt();
        int z = input.nextInt();

        int max = x;
        if(y > max)
            max = y;
        if(z > max)
            max = z;

        System.out.println(max);

        input.close();
    }
}